src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -942,14 +982,10 @@ void Convolution::initDescriptor(const NodeConfig& config) {
     if (isStridedBlobsSupported) {
         createDescriptor({config.inConfs[0].getMemDesc()}, {config.outConfs[0].getMemDesc()});
     }
-    // attr[0] - depthwise, quantize
-    // attr[1] - binary
-    dnnl::primitive_attr attrs[2];
-    auto attrsNum = shouldTryBrgconv ? 2 : 1;
-    setPostOps(attrs[0], MemoryDescUtils::makeDummyShape(getOutputShapeAtPort(0)).getStaticDims(), true);
-    if (shouldTryBrgconv) {
-        setPostOps(attrs[1], MemoryDescUtils::makeDummyShape(getOutputShapeAtPort(0)).getStaticDims(), false);
-    }
+
+    std::array<dnnl::primitive_attr, 2> attrs;
```

Comment:
Didn't get the idea of using array.
It seems it would be more clean to just use a vector, then insert attrs in scope of SetPostOpsAndZeroPoints
Then attrsNum variable is seemed to be unnecessary.